### PR TITLE
Made few typos correct and made it more uniform. (Very minute changes)

### PR DESCRIPTION
**Pull request by Neon-20**
Tiny Changes in Readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,12 +50,12 @@ Seamless Communication depends on 3 libraries developed by Meta.
 fairseq2 is our next-generation open-source library of sequence modeling components that provides researchers and developers with building blocks for machine translation, language modeling, and other sequence generation tasks. All SeamlessM4T models in this repository are powered by fairseq2.
 
 ## [SONAR and BLASER 2.0](https://github.com/facebookresearch/SONAR)
-SONAR, Sentence-level multimOdal and laNguage-Agnostic Representations is a new multilingual and -modal sentence embedding space which outperforms existing sentence embeddings such as LASER3 and LabSE on the xsim and xsim++ multilingual similarity search tasks. SONAR provides text and speech encoders for many languages. SeamlessAlign was mined based on SONAR embeddings.
+SONAR, Sentence-level multimodal and language-Agnostic Representations is a new multi-lingual and modal sentence embedding space which outperforms existing sentence embeddings such as LASER3 and LabSE on the xsim and xsim++ multi-lingual similarity search tasks. SONAR provides text and speech encoders for many languages. SeamlessAlign was mined based on SONAR embeddings.
```
Comment (cndn) [inline README.md:53]: Let's keep "multilingual" as a word and "-modal" (meaning multimodal) otherwise lgtm, thanks for the fix!

**Review by cndn — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -50,12 +50,12 @@ Seamless Communication depends on 3 libraries developed by Meta.
 fairseq2 is our next-generation open-source library of sequence modeling components that provides researchers and developers with building blocks for machine translation, language modeling, and other sequence generation tasks. All SeamlessM4T models in this repository are powered by fairseq2.
 
 ## [SONAR and BLASER 2.0](https://github.com/facebookresearch/SONAR)
-SONAR, Sentence-level multimOdal and laNguage-Agnostic Representations is a new multilingual and -modal sentence embedding space which outperforms existing sentence embeddings such as LASER3 and LabSE on the xsim and xsim++ multilingual similarity search tasks. SONAR provides text and speech encoders for many languages. SeamlessAlign was mined based on SONAR embeddings.
```
Comment (cndn) [inline README.md:53]: I think this is intentional to explain where "SONAR" comes from. And we could keep multilingual as a work and "-modal" meaning multimodal.

Comment (cndn) [reply README.md:53]: we could revert this line change and ship another line. Thanks!

Comment (Neon-20) [reply README.md:53]: sure

**Review by kauterry — APPROVED** [review]

**Pull request merged**